.env

```diff
@@ -0,0 +1,32 @@
+# Used to determine sandbox build type:
```

Comment:
@tzaffi Unrelated to the line:  From PR description, responding to _ Get consensus on the best way to move forward with this PR._

> Medium - Continue with the current PR and leave the V1 Endpoint Problem to a future PR. The current approach requires the step removal/modifications described below.

Having made a brief pass through PRs, the quoted approach makes sense to me.